### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by Tinsane**
Go fmt and docker based integration test.

### CODE REVIEW

File: docker/pg/initdb.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+/usr/lib/postgresql/10/bin/pg_ctl -D $PGDATA -w start
+pgbench -i -s 10 postgres
+pg_dumpall -f /tmp/dump1
+wal-g backup-push $PGDATA
+pkill -9 postgres
```
Comment (x4m) [inline docker/pg/initdb.sh]: Such statements can scare the hell out of DBA's...
but it leaves less logs than pg_ctl stop --immediate, let's leave it

File: docker/pg/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+RUN apt update
+RUN yes | apt install liblzo2-2
+COPY pginstall.sh /tmp
+RUN chmod +x /tmp/pginstall.sh
+RUN /tmp/pginstall.sh | apt install postgresql postgresql-contrib
+ENV PGDATA=/var/lib/postgresql/10/main
```
[the remark this answers is not in the document]
Comment (Tinsane) [reply docker/pg/Dockerfile]: fixed

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+make
+code = $?
+if [ "$code" -ne 0  ]
```
[the remark this answers is not in the document]
Comment (Tinsane) [reply build.sh]: fixed

File: docker/pg/initdb.sh (hunk 1 above)
Comment (Tinsane) [reply docker/pg/initdb.sh]: It's a disaster recovery test. I've just considered "pkill -9" to be one.

**Comment by x4m** [comment]
I have a lot of ideas about what to improve here:
1. Check that .history and .backup files pushes\fetches
2. Check that multiple WAL's are archived by bguploader from archive status
3. Check different compression methods and different PG versions
etc etc

But this test is already important move forward. Thanks!

**Pull request merged**